src/events.h

```diff
@@ -60,13 +60,17 @@ class Events
 
 		// Monster
 		int32_t monsterOnDropLoot = -1;
+		int32_t monsterOnSpawn = -1;
 	};
 
 	public:
 		Events();
 
 		bool load();
 
+		// Monster
+		bool eventMonsterOnSpawn(Monster* monster, const Position& position, bool startup, bool artificial);
```

Comment:
This definition can moved down to the other monster event `eventMonsterOnDropLoot`.